modules/ipi-preparing-reinstall-cluster-bare-metal.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+[id="ipi-preparing-reinstall-cluster-bare-metal_{context}"]
+
+= Preparing to reinstall a cluster on bare metal
+Before you reinstall a cluster on bare metal, you must perform cleanup operations.
+
+.Procedure
+. Remove or reformat the disks for the bootstrap, <control-plane>, and worker nodes. If you are working in a hypervisor environment, you must add any disks you removed.
+. Delete the artifacts that the previous installation generated:
+
```

Comment:
```suggestion
+
```

Attaching the command to the step is more future-proof than using `[start=3]`.

Reply:
Added + to topic.